hail/python/hail/plot/plots.py

```diff
@@ -1450,27 +1451,44 @@ def qq(
     return p
 
 
-@typecheck(pvals=expr_float64, locus=nullable(expr_locus()), title=nullable(str),
-           size=int, hover_fields=nullable(dictof(str, expr_any)), collect_all=bool, n_divisions=int, significance_line=nullable(numeric))
-def manhattan(pvals, locus=None, title=None, size=4, hover_fields=None, collect_all=False, n_divisions=500, significance_line=5e-8) -> Plot:
+@typecheck(pvals=expr_float64,
+           locus=nullable(expr_locus()),
+           title=nullable(str),
+           size=int,
+           hover_fields=nullable(dictof(str, expr_any)),
+           collect_all=nullable(bool),
+           n_divisions=nullable(int),
+           significance_line=nullable(numeric)
+           )
+def manhattan(pvals: 'Float64Expression',
+              locus: 'Optional[LocusExpression]' = None,
+              title: 'Optional[str]' = None,
+              size: int = 4,
+              hover_fields: 'Optional[Dict[str, Expression]]' = None,
+              collect_all: 'Optional[bool]' = None,
+              n_divisions: 'Optional[int]' = 500,
+              significance_line: 'Optional[Union[int, float]]' = 5e-8
```

Comment:
These types don't need to be in strings unless there's a typing circularity, which, in this case, there isn't. Doesn't affect correctness or anything, just a style thing.